src/mjolnir/landmark_database_builder.cc

```diff
@@ -1,8 +1,39 @@
 #include "mjolnir/landmark_database_builder.h"
 #include "mjolnir/util.h"
 
+#include "mjolnir/osmpbfparser.h"
+
 namespace valhalla {
 namespace mjolnir {
+const std::string landmark_db = "landmarks.db";
+
+static const std::unordered_map<std::string, LandmarkType> str_type_map = {
+    {"parking", LandmarkType::parking},
+    {"bench", LandmarkType::bench},
+    {"parking_space", LandmarkType::parking_space},
+    {"place_of_worship", LandmarkType::place_of_worship},
+    {"restaurant", LandmarkType::restaurant},
+    {"waste_basket", LandmarkType::waste_basket},
+    {"bicycle_parking", LandmarkType::bicycle_parking},
```

Comment:
same for this, doesnt seem relevant. maybe you are thinking about pedestrian routing? i suppose its possible that on unnamed trails smaller things like these could be worth using for navigation points?